csfieldguide/templates/interactives/distance-from-stereo-images.html

```diff
@@ -0,0 +1,115 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+<div id="outer-div" class="container">
+  <h3 class="text-center mb-2">{% trans "Getting Distance From Stereo Images" %}</h3>
+  <a data-toggle="collapse" href="#instructions" role="button" aria-expanded="true" aria-controls="instructions">
+    Instructions (hide/show)
+  </a>
+  <div class="collapse show" id="instructions">
+    <p>
+      {% trans "Take two side-by-side pictures of an object." %}
+      {% trans "Make sure the direction of the pictures are parallel, and that the object is between the two camera positions." %}
+      {% trans "Note down the precise distance between the camera positions and the distance between the camera plane and the object plane." %}
+      {% trans "Upload the two pictures and click on the common point of the object in both pictures." %}
+      {% trans "Enter your camera's angle-of-view and the distance between the two camera positions, and click 'Go!' to calculate the distance." %}
+      {% trans "Note that resizing the window will reset the points as this can affect the size of the pictures." %}
+      {% trans "The values are inputted into the equation below." %}
+    </p>
+    <p>
+      {% trans "The lighter in the supplied images was 1 metre away. How close is the calculation?" %}
+    </p>
+
+    $$ {D =\frac{Bx_0}{2tan(\frac{\theta_0}{2})(x_1-x_2)}} $$
+  </div>
+  <div class="row">
+    <div class="col-sm-6">
+      <h4 class="text-center mb-2">{% trans "Left Camera" %}</h4>
+      <canvas id="canvas-left"></canvas>
+      <img id="img-left" src="{% static 'interactives/distance-from-stereo-images/images/lighter-left.jpg' %}">
+    </div>
+    <div class="col-sm-6">
+      <h4 class="text-center mb-2">{% trans "Right Camera" %}</h4>
+      <canvas id="canvas-right"></canvas>
+      <img id="img-right" src="{% static 'interactives/distance-from-stereo-images/images/lighter-right.jpg' %}">
+    </div>
+  </div>
```

Comment:
Could we please replace the images of the lighter with something more generic and appropriate for teenagers, some cultures wouldn't appreciate the lighter. A piece of fruit, like an apple or banana is a good choice. Also if possible try to take against a blank wall so it's less distracting for the reader.

Reply:
Replaced with pictures of an orange and against a plain background.